src/views/SignInTab.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <ion-page>
+        <ion-content
+            :force-overscroll="false"
+            :fullscreen="true"
+        >
+            <ion-header>
+                <h3 class="center">NutriLog SNHU</h3>
+            </ion-header>
+
+            <Authenticator>
+                <template v-slot="{ user }">
+                    <h1>Hello {{ user.username }}!</h1>
+                    <button @click="signOut">Sign Out</button>
+                    <br>
+                    <button @click="deleteUser">Delete User</button>
+                </template>
+            </Authenticator>
+        </ion-content>
+    </ion-page>
+</template>
+
+<script lang="ts">
+import { Authenticator } from '@aws-amplify/ui-vue';
+import { Hub, Auth, I18n } from 'aws-amplify';
+import '@aws-amplify/ui-vue/styles.css';
+import { defineComponent } from 'vue';
+
+import {
+    IonPage,
+    IonContent,
+    IonHeader
+} from '@ionic/vue';
+
+// Typographical changes for the UI
+I18n.putVocabulariesForLanguage('en', {
+    'Sign in': 'Sign In',
+    'Forgot your password? ': 'Reset Password',
+});
+
+export default defineComponent({
+    components: {
+        IonPage,
+        IonContent,
+        IonHeader,
+        Authenticator
+    },
+    setup() {
+        return {
+        }
+    },
+    mounted() {
+        // Listen for Auth events
+        const AuthListener = (data: any) => {
+            switch (data.payload.event) {
+                case 'signIn':
+                    console.log('User signed in.');
```

Comment:
Debug print?

Reply:
I left those there originally to be a clear place to attach the users API to backend. They are now just comments

Reply:
I left those there originally to be a clear place to attach the users API to backend. They are now just comments